### DISCUSSION TITLE
Post Sort Order

### DISCUSSION THREAD

**Discussion by neelakantan-ca** [jpanther/congo#510] [Q&A]
Hi

I am totally new to hugo and this theme. I have gone over the documentation for this theme, and some relevant hugo documentation. I may have missed something so please by patient with me.

I am trying to either reverse the sort order on list pages or use weights to change the order of posts.

I've looked through the following links:

- https://gohugo.io/content-management/front-matter/
- https://gohugo.io/templates/lists/#order-content
- https://github.com/jpanther/congo/issues/313

Could someone please guide me on how I would go about changing the sort order ? 

Also a massive thanks to creator of this theme, it has helped massively in quickly setting up a small website.

**Comment by uschmidt83** [comment]
Hi, you can use [weight](https://gohugo.io/content-management/front-matter/#front-matter-variables) in the front matter.

From the docs:

> **weight**
> used for [ordering your content in lists](https://gohugo.io/templates/lists/). Lower weight gets higher precedence. So content with lower weight will come first. If set, weights should be non-zero, as 0 is interpreted as an unset weight.

**Reply by neelakantan-ca** [reply]
I tried that, it didn't change anything

**Reply by uschmidt83** [reply]
All I can say is that it worked for me. Did you assign a weight to every content page in the list?

**Reply by neelakantan-ca** [reply]
Yup, just double checked, regardless thannks for trying to help 😄
